charts/emissary-ingress/templates/_helpers.tpl

```diff
@@ -45,6 +45,26 @@ Then if the image repository is explicitly set, use "repository:image"
 {{- end -}}
 {{- end -}}
 
+{{/*
+Set the image that should be used for the canary deployment.
+disabled if fullImageOverride is present
+*/}}
+{{- define "ambassador.canaryImage" -}}
+{{- if .Values.image.fullImageOverride }}
+{{- printf "%s" "" -}}
+{{- else if and .Values.canary.image.repository .Values.canary.image.tag -}}
+{{- printf "%s:%s" .Values.canary.image.repository .Values.canary.image.tag -}}
+{{- else if .Values.canary.image.tag -}}
+{{- if hasKey .Values.image "repository" -}}
+{{- printf "%s:%s" .Values.image.repository .Values.canary.image.tag -}}
+{{- else -}}
+{{- printf "%s:%s" "docker.io/datawire/ambassador" .Values.canary.image.tag -}}
```

Comment:
whoops realizing now that this should actually be `docker.io/datawire/emissary`, but that's also wrong above